src/p3.rs

```diff
@@ -22,13 +22,23 @@ impl<F: Field> BaseAir<F> for VMAir {
         11
     }
 }
+type Val = Mersenne31;
+type Challenge = BinomialExtensionField<Val, 3>;
+type ByteHash = Keccak256Hash;
+type FieldHash = SerializingHasher<ByteHash>;
+type MyCompress = CompressionFunctionFromHasher<ByteHash, 2, 32>;
+type ValMmcs = MerkleTreeMmcs<Val, u8, FieldHash, MyCompress, 32>;
+type ChallengeMmcs = ExtensionMmcs<Val, Challenge, ValMmcs>;
+type Challenger = SerializingChallenger32<Val, HashChallenger<u8, ByteHash, 32>>;
+type Pcs = CirclePcs<Val, ValMmcs, ChallengeMmcs>;
+#[allow(dead_code)]
+type MyConfig = StarkConfig<Pcs, Challenge, Challenger>;
```

Comment:
slightly cleaner IMHO to place here, but unsure
`#[allow(dead_code)]` to reduce strange warning of these not being used :shrug: